storybook/.storybook/preview-head.html

```diff
@@ -4,7 +4,25 @@
     height: 400px;
   }
 
+  ol-map {
+    display: block;
+  }
+
   body > #storybook-root > #root-inner {
     height: 95vh;
   }
+
+  #root-inner {
+    flex-direction: column;
+  }
 </style>
+
+<script type="module">
+  import Style from 'ol/style/Style.js';
+  import Fill from 'ol/style/Fill.js';
+  import Stroke from 'ol/style/Stroke.js';
+
+  window.Style = Style;
+  window.Fill = Fill;
+  window.Stroke = Stroke;
```

Comment:
Do you think it would make sense to scope them into a dedicated namespace like `window.openlayersElements.` to avoid conflicts?

Reply:
I'd rather figure out a way to import in a story but otherwise, sure, why not...

Reply:
I know that it is only for Storybook in that particular case, but people like to copy-paste working examples, and some of them could theoretically face some issues because of this, as `Style` and other terms are quite common.

But it's not blocking, and not mandatory to do.

Reply:
To copy-paste, you would not want an additional namespace under `windows`. In turn I added the imports commented-out, as they would be used in actual code.

Also, had to switch to a CDN bundler because apparently the preview script is not bundled in SB build